docs/content/concepts/cryptography/transaction-auth/intent-signing.mdx

```diff
@@ -71,18 +71,18 @@ const signature = await this.sign(intentMessage);
-1.  Applies Blake2b hash to get the 32-byte digest
-1.  Passes the digest to the signing API for each corresponding scheme of the signer. The supported signature schemes are pure Ed25519, ECDSA Secp256k1 and ECDSA Secp256r1. See [Sui Signatures](./signatures.mdx#signature-requirements) for requirements of each scheme.
+1.  Applies Blake2b hash to get the 32-byte digest.
+1.  Passes the digest to the signing API for each corresponding scheme of the signer. The supported signature schemes are pure Ed25519, ECDSA Secp256k1, and ECDSA Secp256r1. See [Sui Signatures](./signatures.mdx#signature-requirements) for requirements of each scheme.
 
 ## Authority signature
 
-The authority signature is created using the protocol key. The data that it commits to is also an intent message `intent || message`. See all available intent scopes [in the source code](https://github.com/MystenLabs/sui/blob/0dc1a38f800fc2d8fabe11477fdef702058cf00d/crates/sui-types/src/intent.rs#L66)
+The authority signature is created using the protocol key. The data that it commits to is also an intent message `intent || message`. See all available intent scopes [in the source code](https://github.com/MystenLabs/sui/blob/0dc1a38f800fc2d8fabe11477fdef702058cf00d/crates/sui-types/src/intent.rs#L66).
 
 ### How to generate proof of possession for an authority
 
-When an authority request to join the network, the protocol public key and its proof of possession (PoP) are required to be submitted. PoP is required to prevent [rogue key attack](https://crypto.stanford.edu/~dabo/pubs/papers/BLSmultisig.html).
+When an authority requests to join the network, the protocol public key and its proof of possession (PoP) are required to be submitted. PoP is required to prevent [rogue key attack](https://crypto.stanford.edu/~dabo/pubs/papers/BLSmultisig.html).
 
-The proof of possession is a BLS signature created using the authority's protocol private key, committed over the following message: `intent || pubkey || address || epoch`. Here `intent` is serialized to `[5, 0, 0]` representing an intent with scope as `Proof of Possession`, version as `V0` and app_id as `Sui`. `pubkey` is the serialized public key bytes of the authority's BLS protocol key. `address` is the account address associated with the authority's account key. `epoch` is serialized to `[0, 0, 0, 0, 0, 0, 0, 0]`.
+The proof of possession is a BLS signature created using the authority protocol private key, committed over the following message: `intent || pubkey || address || epoch`. Here `intent` is serialized to `[5, 0, 0]` representing an intent with scope as `Proof of Possession`, version as `V0`, and app_id as `Sui`. `pubkey` is the serialized public key bytes of the authority BLS protocol key. `address` is the account address associated with the authority account key. `epoch` is serialized to `[0, 0, 0, 0, 0, 0, 0, 0]`.
```

Comment:
```suggestion
The proof of possession is a BLS signature created using the authority protocol private key, committed over the following message: `intent || pubkey || address || epoch`. Here `intent` is serialized to `[5, 0, 0]` representing an intent with scope as `Proof of Possession`, `version` as `V0`, and `app_id` as `Sui`. The `pubkey` value is the serialized public key bytes of the authority BLS protocol key. The `address` value is the account address associated with the authority account key. The `epoch` value is serialized to `[0, 0, 0, 0, 0, 0, 0, 0]`.
```
"The ... value" stuff is personal preference not to begin a sentence with inline code. No reason other than aesthetics.